### DISCUSSION TITLE
MSI Delta 15 (AMD Advantage Laptop)

### DISCUSSION THREAD

**Discussion by NathanTJFA** [YoCodingMonster/OpenFreezeCenter#130] [General]
System: Its the MSI Delta 15 which is one of their AMD advantage laptops. Even though it isn't on the list of supported devices many of the features seem to work/worked properly. 

Presets: The CPU seems to only follow the auto/normal preset even when its set to fast, slow, etc. I modified the config to align it more to the slowest config.

Advanced: The GPU still works perfectly but the CPU fan will be on regardless of what the advanced fan is set to. Therefore I basically just use the Slowest preset for fans off and the Advanced preset for fans on.

Cooler Booster: Works perfectly

Battery Profile: Doesn't work at all.

Monitor: The monitor for the GPU seems to work while the CPU one just copies it (This does not happen on 3.1 where the monitoring works perfectly) Intel 11th gen makes it spit completely random numbers.

CPU Profile: This one used to work perfectly but recently has been not working for basically no reason I've posted an issue about it in the Issues tab: https://github.com/YoCodingMonster/OpenFreezeCenter/issues/126

**Comment by Ritul13** [comment]
I want you to help me to my problem i can not able to start or run it when i extracted it in desktop and write the code that is given in readme file.... I am using ARCH linux distro GRUDA
